### PR TITLE
Lookup user handles from Twitter.

### LINKED ISSUE #79: Needed: Map missing user_id -> user_handle with remote services
This will improve `followers.txt`, `following.txt`, `DMs.md` where currently many handles are missing.

Suggestions and initial work for how to retrieve these handles has happened in several places recently. Thank you!

Ping:  @flauschzelle, @lenaschimmel, @press-rouch, @n1ckfg (but of course anyone can contribute)

[Edit: the PRs that were mentioned are now merged]

### PR DESCRIPTION
**Pull request by press-rouch**
Closes #79 

We are getting the full metadata and discarding most of it, so we could be doing a lot more with this.

There is a slight inefficiency due to creating new sessions for each of the following, follower, and DM lookups, but I wanted to keep the changes to a minimum for now.

Following and followers tested. DMs not tested as I don't have any.

### CODE REVIEW

**Review by JoeTorrey — COMMENTED** [review]
Cool thanks 🙏

**Review by JoeTorrey — APPROVED** [review]
Thanks a lot for all the advice and help you guys it’s greatly appreciated

**Review by JoeTorrey — COMMENTED** [review]
Thanks so much and I hope you’re doing well 🙏🤞🏽🙌

**Comment by timhutton** [comment]
image

**Comment by timhutton** [comment]
(Have blocked the spambot 'JoeTorrey'. But can't work out how to delete the reviews.)

File: parser.py [hunk 1]
```diff
@@ -390,30 +432,48 @@ def main():
...
     following = []
     following_json = read_json_from_js_file(os.path.join(data_folder, 'following.js'))
+    following_ids = []
     for follow in following_json:
         if 'following' in follow and 'accountId' in follow['following']:
-            id = follow['following']['accountId']
-            handle = users[id].handle if id in users else '~unknown~handle~'
-            following.append(handle + ' ' + user_id_URL.format(id))
+            following_ids.append(follow['following']['accountId'])
+    lookup_users(following_ids, users)
```
Comment (timhutton) [inline parser.py:439]: Need to confirm with user before doing any internet access.

**Comment by timhutton** [comment]
Works beautifully on my followers, followings and DMs. Nice work @press-rouch

**Review by timhutton — APPROVED** [review]

File: parser.py (hunk 1 above)
Comment (press-rouch) [reply parser.py:445]: Added in changeset e6cb01b.

**Review by timhutton — APPROVED** [review]

**Comment by timhutton** [comment]
image

**Pull request merged**